portfolio/src/main/webapp/script.js

```diff
@@ -16,3 +16,16 @@ window.addEventListener('DOMContentLoaded', (event) => {
       }
   }
 });
+
+function getComments() {
+    fetch('/data').then(response => response.json()).then((comments) => {
+        
+        const commentElement = document.getElementById('comment-container');
+        commentElement.innerHTML = '';
+
+        for(var i = 0; i < comments.length; i++) {
```

Comment:
- space between for and (
- prefer const/let over var
- actually, here is a perfect time to use a JS for-of loop e.g.

```
for (let comment of comments) {
  commentElement.append(comment + " ");
}
```